relayer/strategies.go

```diff
@@ -119,6 +150,8 @@ func relayerStartEventProcessor(
 				log,
 				p.src.pathEnd,
 				p.dst.pathEnd,
+				packetObservedCounter,
+				packetRelayedCounter,
```

Comment:
Instead of 2 args here, could be refactored to accept a single "metrics" interface. This would also make it more extensible in the future. You can extend the interface instead of method args. My guess is it's likely more metrics will want to be captured in the future here.

Reply:
This is a good call. It required moving the `PrometheusMetrics` struct out of the relayer package due to an import cycle, but seemed okay to move it to the processor package since that is where it's used.